baseapp/baseapp_test.go

```diff
@@ -221,11 +221,12 @@ const msgType = "testUpdatePowerTx"
 func (tx testUpdatePowerTx) Type() string                            { return msgType }
 func (tx testUpdatePowerTx) Get(key interface{}) (value interface{}) { return nil }
 func (tx testUpdatePowerTx) GetMsg() sdk.Msg                         { return tx }
-func (tx testUpdatePowerTx) GetSignBytes() []byte                    { return nil }
+func (tx testUpdatePowerTx) GetSignBytes(sdk.Context) []byte         { return nil }
```

Comment:
As per my comment on the Conversation thread, I think we should keep GetSignBytes() as is.